simplesat/request.py

```diff
@@ -8,7 +8,8 @@
 class JobType(Enum):
     install = 1
     remove = 2
-    update = 3
+    soft_update = 3
```

Comment:
This makes 0.4.0 backward incompatible. This is fine, but can you make 2 small PRs on  edm master and maintenance/0.6.x to lock the current satsolver versions to `>= 0.3.1, < 0.4.0` ?